app/src/test/java/org/simple/clinic/home/overdue/OverdueUiRendererTest.kt

```diff
@@ -16,7 +16,7 @@ class OverdueUiRendererTest {
   private val defaultModel = OverdueModel.create()
 
   @Test
-  fun `when overdue appointments are loaded and overdue sections is enabled, then show overdue appointments list and overdue count`() {
+  fun `when overdue appointments are loaded and overdue sections is enabled, then show overdue appointments list, overdue count and hide progress`() {
```

Comment:
I feel the test name is getting longer, should we instead call it "render all patients overdue list"?